### ISSUE TITLE
Add additional info to scheduler common commands

### ISSUE THREAD

**Issue by Carldkennedy** [rcgsheffield/sheffield_hpc#1613]
https://docs.hpc.shef.ac.uk/en/latest/referenceinfo/scheduler/SLURM/Common-commands/index.html
https://docs.hpc.shef.ac.uk/en/latest/referenceinfo/scheduler/SGE/Common-commands/index.html

Add info about man page navigation etc

By default, man uses less as its pager. In less, you can use:
Documentation is available on the system using the command
man less

**Comment by jkwmoore** [comment]
I'm not sure we need to go to the level of detail where we're directing people on how to use man?

We could do so as a pop-up? Wouldn't "take up space" on the pages if we used them: https://sphinx-hoverxref.readthedocs.io/en/latest/

**Comment by Carldkennedy** [comment]
It was something @willfurnass thought might be useful to users for investigating how to investigate commands available on the clusters.

**Comment by willfurnass** [comment]
Yes, I think that pointing advanced users at man pages is a good way of keeping our instructions concise.

Very basic guide to `man` could go in the FAQ.

Tooltips eh?   Hmmmmmm.
